calculator.py

```diff
@@ -3,11 +3,27 @@
 class Calculator(object):
 
   def add(self, x, y):
-    return x + y
+    return x + y 
+
+  def sub(self, x, y):
+    return x - y
+
+  def div(self, x, y):
+    return x / y
 
   def multiply(self, x, y):
     return x * y
+ 
+  def add(self, x, y):
+    return x + y 
+
+  def mod(self, x, y):
```

Comment:
Why do we need two ```add``` functions?

Reply:
Basically, to perform program as many times as user wants we defined function.add is a function name.Function add will take two integers as input and return the sum.To perform math operation we used + symbol.

Reply:
Hey @mugdhadalal Please read question carefully. I agree with your explaination but if you see you have defined ```add``` function for two times(line 17 and line 5)
- can't we remove one?